guides/common/modules/proc_creating-a-resource-quota.adoc

```diff
@@ -2,9 +2,25 @@
 = Creating a resource quota
 
 Create a resource quota to limit the resource consumption of {Project} users.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-resource-quota[].
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Configure* > *Resource Quotas*.
 . Click *Create resource quota*.
 . Specify the name, CPU cores, memory, and disk space.
 . Click *Create resource quota* to submit the resource quota to {Project}.
+
+[id="cli-creating-a-resource-quota"]
+.CLI procedure
+* Create a resource quota:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer resource-quota create \
```

Comment:
If you create or update a resource quota, you can also set the '--description' option.

Reply:
Added.